Resources/doc/reference/introduction.rst

```diff
@@ -73,6 +73,9 @@ By default the ``SonataPageBundle`` is shipped with core block services:
 
  - sonata.page.block.container      : Block container
  - sonata.page.block.children_pages : Render a navigation panel
+ - sonata.page.block.breadcrumb     : Render a breadcrumb
```

Comment:
@stphnpt : use `git add --patch` to commit different parts of a file separately. It's also a great tool to review your own code before committing it.